src/logic/calculate.js

```diff
@@ -0,0 +1,31 @@
+import Symbols from '../symbols';
+import operate from './operate';
+
+const calculate = (calculator, buttonName) => {
+  const { next, total } = calculator;
+
```

Comment:
- `calculator data object` contains three properties: `total`, `next` and `operation` [ [instructions](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/totalNextOperate.png) ]

Reply:
from your destructuring, seems to have 2  properties👨

Reply:
If I destructure a property that I have not yet used then the linter - would complain, that's why I skipped it for now.